### DISCUSSION TITLE
Project 'Insights' does not match actual project numbers

### DISCUSSION THREAD

**Discussion by allieculp** [community/community#53919] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Having trouble figuring out why our Project Insights do not match the data in our actual Project view.

Project: https://github.com/orgs/openxla/projects/13
Insight Chart: https://github.com/orgs/openxla/projects/13/insights/8

Example, in the chart it says we have 13 inbox items today, where we only have 4 in our Project view. Thanks for the help!!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jess-fortier** [comment]
I am experiencing this as well. I can't figure it out! Unsure if the "Invalid Date undefined. J u" text I'm seeing on-hover is part of the underlying problem or just yet another separate problem - there have been so many! - that I should ignore as a red herring.

Seems a bit bold to call this feature "Insights" when its data isn't accurate enough to be insightful :( 

In the table view of our backlog we have 16 items totaling 54.5 story points across the "In progress", "Blocked", "Review", and "Done" statuses: 
![image](https://github.com/community/community/assets/793901/87765728-6a51-4f4d-a6ff-88185f40c64f)

And the filtered version of our "Burn up / CFD" which gives 49 storypoints across "To do", "In progress" and "Blocked":
![image](https://github.com/community/community/assets/793901/5e34975e-46f1-435e-a3af-88ea8ff0bb27)

**Comment by dwainm** [comment]
I can also confirm this; the data for the status on the chart doesn't match the column count.

Screenshots taken on 11 August:
<img width="211" alt="Screenshot 2023-08-11 at 11 43 05" src="https://github.com/community/community/assets/1713474/a3cf6036-e369-4711-b2ee-cf23b84ecedf">

<img width="354" alt="Screenshot 2023-08-11 at 11 43 18" src="https://github.com/community/community/assets/1713474/59c3d955-78e0-4295-be7e-ab3d0220f1d0">

<img width="410" alt="Screenshot 2023-08-11 at 11 45 24" src="https://github.com/community/community/assets/1713474/85974821-17a7-4028-a1c4-5bdc329a8110">

It might be that the report is not respecting the sum of a field value there.
